.github/workflows/push_pr.yml

```diff
@@ -46,6 +47,31 @@ jobs:
               echo "No unit tests found for $chart"
             fi
           done
+      - name: Setup Minikube
+        uses: manusa/actions-setup-minikube@v2.4.3
+        if: steps.list-changed.outputs.changed == 'true'
+        with:
+          minikube version: v1.20.0
```

Comment:
nit: i think there could be some newer versions

Reply:
you are right, I don't even know why we hard code that, I'll bump it or remove it. Thanks for noticing

Reply:
we cannot remove it since it is required, but we can bump it